client/state/support/selectors.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param  {Object} state Global state tree
+ * @return {Object}       The support section of the state
+ */
+export function getSupportState( state ) {
```

Comment:
This selector function seems a bit odd to me. For something this general I think it's better to just do `state.support`. The name is also not that clear. I think we should have individual functions like `getSupportUser` or `getSupportToken`.